src/renderer/components/AircraftSection/LiveryConversion/index.tsx

```diff
@@ -0,0 +1,249 @@
+
+    const handleCancel = () => setShowList(false);
+
+    const handleConfirm = () => {
+        selectedLiveries.forEach((livery) => {
+            LiveryConversion.convertLivery(livery).then(() => {
+                store.dispatch<LiveryAction>({
+                    type: actionTypes.SET_LIVERY_STATE,
+                    payload: {
+                        livery,
+                        state: LiveryState.CONVERTED,
+                    },
+                });
+            }).catch((error) => {
+                store.dispatch<LiveryAction>({
+                    type: actionTypes.SET_LIVERY_STATE,
+                    payload: {
+                        livery,
+                        state: LiveryState.ERROR_DURING_CONVERSION,
+                        error
+                    },
+                });
+            });
+        });
+    };
+
+    return (
+        <div className="flex flex-col gap-y-5 bg-red-600 px-5 py-5 rounded-lg">
+            <div className="flex flex-row justify-between items-center">
+                <div className="flex flex-col">
+                    <span className="text-2xl text-white font-semibold">
+                        {liveries.length > 1 ? `${liveries.length} incompatible liveries are` : '1 incompatible livery is'} present in your Community folder.
+                    </span>
+                    <span className="text-xl text-white font-semibold mt-2.5">
+                        Do you wish to convert them?
+                    </span>
+                    <span className="text-lg text-red-200 font-semibold">
+                        Warning: it is your responsibility to make sure the livery authors allow conversion.
+                    </span>
+                </div>
+                <div className="flex flex-row justify-center items-center">
+                    {showList &&
+                        <button
+                            className={`w-32 bg-red-700 bg-opacity-40 mr-4 transition-colors duration-200 hover:bg-red-700 px-6 py-2 text-2xl rounded-lg text-white`}
+                            onClick={allLiveriesConverted ? handleDone : handleCancel}
+                        >
+                            {allLiveriesConverted ? 'Done' : 'Cancel'}
+                        </button>
+                    }
+                    {!showList &&
+                        <a className="text-xl text-white font-semibold mr-3.5" onClick={handleDontAskAgain}>
+                            Don't Ask Again
+                        </a>
+                    }
+                    <button
+                        disabled={buttonDisabled()}
+                        className={`w-40 bg-red-500 transition-colors duration-200 hover:bg-red-700 px-6 py-2 text-2xl rounded-lg text-white ` + (buttonDisabled() ? 'pointer-events-none opacity-60' : '')}
+                        onClick={showList ? handleConfirm : handleConvert}
+                    >
+                        {showList ? 'Confirm' : 'Convert...'}
```

Comment:
The `Convert...` throws me off and looks to me like the text is cut off on the button.

```suggestion
                        {showList ? 'Confirm' : 'Convert'}
```